packages/react/src/components/Table/TableHead/TableHead.jsx

```diff
@@ -803,6 +803,7 @@ const TableHead = ({
             isFilterable: !isNil(column.filter),
             isMultiselect: column.filter?.isMultiselect,
             width: column.width,
+            customInput: column.filter?.customInput,
```

Comment:
You're adding a new prop to the column that is currently not expected. Please update the `components/Table/TablePropTypes.js` file with this new "customInput" filter prop